_sass/_components--button.scss

```diff
@@ -0,0 +1,3 @@
+.btn-lg {
+  padding: $spacer*1.25 $spacer*3.75;
```

Comment:
There is "$btn-padding-y-lg, $btn-padding-x-lg, $font-size-lg, $btn-border-radius-lg" variables for btn-lg